superset-frontend/package.json

```diff
@@ -126,6 +126,7 @@
     "d3-scale": "^2.1.2",
     "dom-to-image": "^2.6.0",
     "emotion-rgba": "0.0.9",
+    "eslint-plugin-literal-colors": "file:eslint/eslint-plugin-literal-colors",
```

Comment:
use relative directory. An additional suggestion, use a more generic directory name, for example: `build-tools` or other wises.

```
    "eslint-plugin-literal-colors": "./file:eslint/eslint-plugin-literal-colors",
```

Reply:
I agree with @zhaoyongjie about the name. I personally prefer `tools` to leave room for other tools that are not build-related.

Reply:
For now, I moved it to `buildtools`

Reply:
I'm still not sure about this name. Since this is a top-level folder, I would really like to use the opportunity to name it more generically. In the future, we could have sub-folders for `build`, `formatting`, etc. If not, at least let's rename it to `build-tools` to follow the same plugin naming convention. 

I'm curious for @rusackas input here as well.

Reply:
Ok agreed. Moved it to `tools`